daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -360,7 +361,7 @@ private[lf] object Pretty {
   def prettyValue(verbose: Boolean)(v: Value[ContractId]): Doc =
     v match {
       case ValueInt64(i) => str(i)
-      case ValueNumeric(d) => str(d)
+      case ValueNumeric(d) => text(data.Numeric.toString(d))
```

Comment:
does this change anything? I wasn’t expecting to see a change to `Numeric` in here. I guess the pretty printer doesn’t matter much so no issues with keeping it if it changes it in a useful way.

Reply:
I just realized the Numeric are printed in a consistent way with LF builtin to TO_TEXT_NUMERIC.
This is used in the output of LF Repl wish it is match with EXPETED.ledger.